code/modules/awaymissions/corpse.dm

```diff
@@ -12,6 +12,8 @@
 	var/instant = FALSE	//fires on New
 	var/flavour_text = "The mapper forgot to set this!"
 	var/faction = null
+	var/permanent = FALSE	//Set to true for infinite mob spawn I guess.
```

Comment:
I'll probably do both tbh. The permanent var will say if it disappears after it is used up, and the uses var would control the actual spawning.